### DISCUSSION TITLE
zahlendes Familienmitglied bei Normalzahlern

### DISCUSSION THREAD

**Discussion by tolot27** [openjverein/jverein#341] [General]
Wir haben bei uns einen Familienbeitrag (65 €/a) und einen Beitrag für Einzelpersonen (30 €/a). Manche Alleinerziehende entscheiden sich, zweimal eine Einzelmitgliedschaft abzuschließen, um 5 € zu sparen. Wie bekomme ich es nun hin, dass ich beide zu einer Familie zusammenführe und die Einzelbeiträge über eine Lastschrift vom Hauptzahler einziehen kann? Die Option "Zahlendes Familienmitglied" gibt es ja nur bei der Beitragsgruppe "Familienmitglied" und nirgends sonst.
Ich habe versucht, eine weitere Beitragsgruppe für ein "Familieneinzelmitglied" anzulegen. Dann darf ich jedoch keinen Beitrag > 0 angeben.

Geht das nur über Zusatzbeiträge bei dem Familieneinzelmitglied?

**Comment by lenilsas** [comment]
Im Nigthly Build ist es möglich auch Familienangehörigen einen Beitrag zuzuweisen.

**Reply by tolot27** [reply]
Ah stimmt, hatte ich irgendwo kürzlich gelesen (#286). Danke, funktioniert! 😄
